### DISCUSSION TITLE
i3bar: show workspace buttons for all outputs

### DISCUSSION THREAD

**Discussion by mtvec** [i3/i3#4820] [Q&A] [answered]
I have a dual monitor setup and would like to have a single `i3bar` on my primary screen. This is easy enough using `output primary`. However, this only shows the workspace buttons for the workspaces on the primary screen. This essentially forces me to add an `i3bar` on the secondary monitor as well just to be able to keep track of its workspaces.

Is there an option to make `i3bar` show buttons for *all* workspaces? If not, is this a feature that could be considered?

**Comment by Airblader** [comment, marked as answer]
This will be possible when https://github.com/i3/i3/pull/4311 lands.

**Reply by mtvec** [reply]
Ok, thanks!

Any known workarounds that currently work? I wonder how other people work on such a setup. Or do most people just an `i3bar` on all screens?

**Reply by Airblader** [reply]
With i3bar you cannot get all workspaces on the instance of a single output. You could switch to something like polybar; no idea if it can do that, but I'd blindly assume it can.

> Or do most people just an i3bar on all screens?

I think so; at least I do, anyway.
